scapy/sessions.py

```diff
@@ -326,14 +327,18 @@ def _process_packet(self, pkt):
             packet = tcp_reassemble(bytes(data), metadata)
         # Stack the result on top of the previous frames
         if packet:
+            if metadata["seq"] is not None:
+                pkt[TCP].seq = metadata["seq"]
             data.clear()
             metadata.clear()
             del self.tcp_frags[ident]
             pay.underlayer.remove_payload()
             if IP in pkt:
                 pkt[IP].len = None
                 pkt[IP].chksum = None
-            return pkt / packet
+            pkt = pkt / packet
+            pkt.wirelen = len(raw(pkt))
```

Comment:
This is technically incorrect since this is not the wirelen, but you can do
```
pkt.wirelen = None
```
to have the same effect